readme.md

```diff
@@ -14,6 +14,22 @@ Running tests requires [phpunit](https://phpunit.de/).
 TEST=1 phpunit
 ```
 
+# Usage
+
+Installing this plugin provides a new Intercom settings page, which allows you to configure your app id and secure mode secret:
+
+<img src="https://raw.githubusercontent.com/intercom/intercom-wordpress/master/screenshots/settings.png"/>
+
+Once filled out, the Intercom widget will automatically appear:
+
+<img src="https://raw.githubusercontent.com/intercom/intercom-wordpress/master/screenshots/widget.png"/>
+
+# Users
+
+If a `$current_user` is present, we use their email as an identifier in the widget.
+
+Otherwise the widget operates in [Acquire mode](https://www.intercom.io/live-chat) (if available). This allows you to chat to anonymous users.
```

Comment:
s/to chat to anonymous users/to talk with anonymous visitors on your Wordpress site/ ? 

More similar to the marketing blurb here: https://www.intercom.io/live-chat